### DISCUSSION TITLE
Backup failed because: The dump process failed with exitcode 2 : Misuse of shell builtins : mysqldump: Got error: 2004: "Can't create TCP/IP socket (10106)" when trying to connect

### DISCUSSION THREAD

**Discussion by Razwanullah** [spatie/laravel-backup#1512] [General]
Hello
When I want to backup via command line everything works fine via command
php artisan backup:run
or
php artisan backup:run --only-db
or
php artisan backup:run --only-files
But when I execute the command from the controller, I have the following error:
Backup failed because: The dump process failed with exitcode 2 : Misuse of shell builtins : mysqldump: Got error: 2004: "Can't create TCP/IP socket (10106)" when trying to connect
note: this error not shown with php artisan backup:run --only-files command
please any help!!

**Comment by PaolaRuby** [comment]
[Misuse of shell builtins](https://www.google.com/search?q=Misuse+of+shell+builtins+%3A+mysqldump%3A+Got+error%3A+2004%3A+%22Can%27t+create+TCP%2FIP+socket+(10106)%22+when+trying+to+connect&oq=Misuse+of+shell+builtins+%3A+mysqldump%3A+Got+error%3A+2004%3A+%22Can%27t+create+TCP%2FIP+socket+(10106)%22+when+trying+to+connect&aqs=chrome..69i57.695j0j1&sourceid=chrome&ie=UTF-8)
